tsconfig.json

```diff
@@ -0,0 +1,11 @@
+{
+  "compilerOptions": {
+    "target": "es5",                                /* Specify ECMAScript target version: 'ES3' (default), 'ES5', 'ES2015', 'ES2016', 'ES2017', 'ES2018', 'ES2019', 'ES2020', or 'ESNEXT'. */
+    "module": "commonjs",                           /* Specify module code generation: 'none', 'commonjs', 'amd', 'system', 'umd', 'es2015', 'es2020', or 'ESNext'. */
+    "allowJs": true,                                /* Allow javascript files to be compiled. */
+    "strict": true,                                 /* Enable all strict type-checking options. */
+    "esModuleInterop": true,                        /* Enables emit interoperability between CommonJS and ES Modules via creation of namespace objects for all imports. Implies 'allowSyntheticDefaultImports'. */
+    "skipLibCheck": true,                           /* Skip type checking of declaration files. */
+    "forceConsistentCasingInFileNames": true        /* Disallow inconsistently-cased references to the same file. */
+  }
```

Comment:
do we need a 

```
"exclude": [
    "node_modules"
]
```

to make sure that they're not compiled to typescript as well?  If the answer is no, then I'll remove that from my projects.

also hoping that we can soon target ES2015 -- it makes the generated files a lot smaller and faster.  On a commercial company that I work with that uses VexFlow, we dropped ES5 support 18 months ago (Internet Explorer, etc.) and there have been no complaints.  But who knows who is importing vexflow and still needs to support legacy browsers.

Reply:
I have given it a try. And I see no changes in size of the j.s